### ISSUE TITLE
[Bug] The path cannot be successfully previewed if it is in Chinese (CSS samples and images cannot be displayed)

### ISSUE THREAD

**Issue by williamlorder** [quarto-dev/quarto-cli#235]
When I upgrade to the latest version 0.2.384, if my path and file name are in Chinese. then I can't preview it successfully (everything works fine after reverting back to 0.2.360). The images and CSS styles disappear. Here is the error that pops up when previewing.
```bash
Output created: 未命名.html

Watching files for changes
Browse at http://localhost:7348/
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/tippy.css (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/quarto-syntax-highlighting.css (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/bootstrap/bootstrap-icons.css (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/bootstrap/bootstrap.min.css (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/clipboard/clipboard.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/quarto.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/popper.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/tippy.umd.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/bootstrap/bootstrap.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/anchor.min.js (404: Not Found)
  /%E5%9B%BE%E7%89%87/nnet_ene.svg (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/clipboard/clipboard.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/quarto.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/popper.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/tippy.umd.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/anchor.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/tippy.css (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/quarto-html/quarto-syntax-highlighting.css (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/bootstrap/bootstrap.min.js (404: Not Found)
  /%E6%9C%AA%E5%91%BD%E5%90%8D_files/libs/bootstrap/bootstrap-icons.css (404: Not Found
```

**Comment by jjallaire** [comment]
This is likely due to a change in our internal webserver library -- should be easy to resolve, more soon!

**Comment by jjallaire** [comment]
This should be resolved here: https://github.com/quarto-dev/quarto-cli/commit/272e82b655f713631d0a9ddc9655002761867c98 (and available in v0.2.385)

Thanks for the report!
